docs/_extra/api-reference/schemas/organization.yaml

```diff
@@ -2,11 +2,15 @@ Organization:
   type: object
   required:
     - id
+    - isDefault
     - logo
     - name
   properties:
     id:
       type: string
+    isDefault:
```

Comment:
I'm not married to this property name; I wanted to avoid `default` because it's a reserved word in JavaScript (and generally a gross name for things in any language).

Reply:
With the exception of selectors, I believe we favour `snake_case` in keys in API responses. eg. `user_info` and `display_name` in the `/api/annotations/{id}` and `/api/search` responses.

`default` is a reserved word for _identifiers_ in JS, but allowed for property names (same as _for_, _delete_ etc.), so I don't think this would pose too much of a problem.

I don't have a particularly strong feeling on `is_default` vs. `default`, except that we perhaps do have some precedence for the later in existing annotation responses `hidden`, `flagged`.